build/commands/scripts/commands.js

```diff
@@ -135,6 +132,28 @@ program
   .option('--gemini_sandbox_client_secret <gemini_sandbox_client_secret>')
   .option('--gemini_sandbox_fee_address <gemini_sandbox_fee_address>')
   .option('--gemini_sandbox_oauth_url <gemini_sandbox_oauth_url>')
+  .option('--gn <arg>', 'Additional gn args, in the form <key>:<value>', collect, [])
+  .option('--ignore_compile_failure', 'Keep compiling regardless of error')
+  .option('--is_asan', 'is asan enabled')
+  .option('--last_chrome_installer <last_chrome_installer>', 'folder contains previous version uncompressed chrome.7z pack file. This folder should be in out dir.')
+  .option('--mac_installer_signing_identifier <id>', 'The identifier to use for signing installers')
+  .option('--mac_signing_identifier <id>', 'The identifier to use for signing')
+  .option('--mac_signing_keychain <keychain>', 'The identifier to use for signing', 'login')
+  .option('--ninja <opt>', 'Additional Ninja command-line options, in the form <key>:<value>', collect, [])
+  .option('--no_gn_gen', 'Build without running gn gen')
+  .option('--notarize', 'notarize targets that support it with Apple')
+  .option('--offline', 'use offline mode for RBE')
+  .option('--prepare_only', 'Do not build targets, but prepare everything (build redirect_cc, update branding, etc.)')
+  .option('--skip_signing', 'skip signing binaries')
+  .option('--tag_ap <ap>', 'ap for stub/standalone installer')
+  .option('--tag_installdataindex <index>', 'installdataindex for stub/standalone installer')
+  .option('--target <target>', 'Comma-separated list of targets to build, instead of the default browser target')
+  .option('--target_android_base <target_android_base>', 'target Android SDK level for apk or aab (classic, modern, mono)', 'classic')
+  .option('--target_android_output_format <target_android_output_format>', 'target Android output format (apk, aab)', 'aab')
```

Comment:
this is the only thing that will be different after this PR is merged. It's Android-specific, so cc @SergeyZhukovsky 

Before this unification, this flag was empty by default in the `build` command but defaulted to `aab` in the `create_dist` command. Now, it defaults to `aab` in both the `build` and `create_dist` commands.

It's set in CI everywhere, so we might consider leaving it empty for local development if that's more convenient.

Reply:
cc @samartnik @SergeyZhukovsky ^

should we have `--target_android_output_format` default to `aab` in `build/create_dist` command or should we default it to empty string, which will internally fallback to `apk`?

Reply:
Yes, `aab` in both `build` and `create_dist` commands is ok. `aab` is primary format now, `apk` is for devs, which they can set manually for local builds.